go/plan.ps1

```diff
@@ -0,0 +1,31 @@
+$pkg_name="go"
+$pkg_origin="core"
+$pkg_version="1.13.1"
+$pkg_description="Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
+$pkg_upstream_url="https://golang.org/"
+$pkg_license="BSD"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_source="https://dl.google.com/go/go$pkg_version.windows-amd64.msi"
+$pkg_shasum="ee6ff4f31766096e6ff00bdd855e007c473cc9080f3588d7be0c5ec0e0b85578"
+$pkg_deps=@("core/visual-cpp-redist-2015", "core/visual-cpp-build-tools-2015")
```

Comment:
I actually just verified that we don't need any of those dependencies. Deleting them now! 💯